src/libs/LoginUtils.ts

```diff
@@ -75,11 +79,43 @@ function areEmailsFromSamePrivateDomain(email1: string, email2: string): boolean
     return Str.extractEmailDomain(email1).toLowerCase() === Str.extractEmailDomain(email2).toLowerCase();
 }
 
+function postSAMLLogin(body: FormData): Promise<Response | void> {
+    return fetch(CONFIG.EXPENSIFY.SAML_URL, {
+        method: CONST.NETWORK.METHOD.POST,
+        body,
+        credentials: 'omit',
+    }).then((response) => {
+        if (!response.ok) {
+            throw new Error('An error occurred while logging in. Please try again');
+        }
+        return response.json() as Promise<Response>;
+    });
+}
+
+function handleSAMLLoginError(errorMessage: string, cleanSignInData: boolean) {
+    if (cleanSignInData) {
```

Comment:
NAB but this almost looks like the same variable name
```suggestion
function handleSAMLLoginError(errorMessage: string, shouldClearSignInData: boolean) {
    if (shouldClearSignInData) {
```